src/python/national_voter_file/us_states/mi/transformer.py

```diff
@@ -297,7 +299,7 @@ def extract_mailing_address(self, input_dict):
         ])
         mail_addr_dict = {}
         if address_str.strip():
-            usaddress_dict, usaddress_type = self.usaddress_tag(address_str)
+            usaddress_dict, usaddress_type = self.usaddress_tag(address_str)[0]
```

Comment:
only the first value is ever used, so i decided to just get the first of the two.